en/application-packages.html

```diff
@@ -96,13 +96,15 @@ <h3 id="rollback">Rollback</h3>
 <h3 id="file-distribution">File distribution</h3>
 <p>
   The application package can have components and other large files.
-  At <em>vespa prepare</em>, these files are distributed to the nodes:
+  When an app is deployed (or more precisely, when the app is <em>activated</em>),
+  these files are distributed to the nodes:
```

Comment:
I learned something! I always thought prepare _distributed_ files to nodes, and activate _enabled_ them for use, this clarifies

Reply:
This is how it was initially, because activate should be fast and reliable, but at some point something happened ...